packages/contracts-bedrock/tasks/genesis-l2.ts

```diff
@@ -262,17 +429,28 @@ task('genesis-l2', 'create a genesis config')
       const allocAddr = name === 'LegacyERC20ETH' ? addr : toCodeAddr(addr)
```

Comment:
Because this is the only one not behind a proxy?

Reply:
Yes that is why